test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -102,7 +102,13 @@ parseAnalyseAndReturnError(string const& _source, bool _reportWarnings = false,
 		if (success)
 			if (!StaticAnalyzer(errors).analyze(*sourceUnit))
 				success = false;
-		if (errors.size() > 1 && !_allowMultipleErrors)
+		int errorCount = 0;
+		for (auto const& currentError: errors)
+		{
+			if (currentError->type() != Error::Type::Warning)
```

Comment:
I've gone ahead and silenced all the unused warnings in those tests.  It's mostly un-naming parameters or adding meaningless statements to code blocks.  A little ugly to me, but not bad.